assets/sass/legacy/_main.scss

```diff
@@ -60,6 +60,24 @@ table.src tr td.src {
   user-select: none;
 }
 
+/* profile */
+.local-storage-fields li.list-group-item{
```

Comment:
Would this be a good time to define the styles of the profile page in a new SASS CSS file instead of editing the legacy CSS code?